CHANGES.md

```diff
@@ -1,3 +1,33 @@
+# grmtools 0.13.0 (XXXX-XX-XX)
+
+## Breaking changes
+
+* `cfgrammar::yacc::grammar::YaccGrammarSpan` now returns `Option<Span>` rather
+  than `Option<&Span>`.
+
+* `cfgrammar::yacc::ast::{Production, Symbol}` no longer derive `Eq` and
+  `PartialEq`. Since both now carry a `Span`, it's easy to confuse "two
```

Comment:
Looks like I missed it in the commit message :frown:, `Symbol` also no longer derives `Hash` for the same reason.

Reply:
Fixed in 1547ee1.